src/lib/orionld/serviceRoutines/orionldGetVersion.cpp

```diff
@@ -39,7 +39,14 @@ extern "C"
 #include "orionld/common/branchName.h"                         // ORIONLD_BRANCH
 #include "orionld/serviceRoutines/orionldGetVersion.h"         // Own Interface
 
-
+#include "/usr/include/boost/version.hpp"                      // BOOST_LIB_VERSION
+#include "/usr/local/include/microhttpd.h"                     // MHD_VERSION (returns a number)
+#include "/usr/include/openssl/opensslv.h"                     // OPENSSL_VERSION_NUMBER
+#include "/usr/local/include/mongo/version.h"                  // MONGOCLIENT_VERSION
+#include "/usr/include/rapidjson/rapidjson.h"                  // RAPIDJSON_VERSION_STRING
```

Comment:
Include system headers with "<>", and remove /usr/include" from the path

Reply:
E.g. #include <boost/version.hpp>

Reply:
Fixed in 0207b286534bbab5cbc493a919e737d6f9ab29fa

Reply:
Fixed in 0207b28